android/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,13 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="io.customer.reactnative.sdk">
 
+    <application>
+        <service
+            android:name=".messagingpush.CustomerIORNFirebaseMessagingService"
+            android:exported="false">
+            <intent-filter android:priority="-1">
```

Comment:
What if a customer doesn't want this behavior in their app? They may want to provide their own `FirebaseMessagingService` and if the customer forgets to set the `android:priority` property or gives it an incorrect value, then the customer could become frustrated wondering why their service isn't being used. This sounds like a risky change to me. 

I also recommend that we add a comment to this file explaining why we added this block of code.

Reply:
Comments are already added to `CustomerIORNFirebaseMessagingService` class. For changes in manifest, I don't understand the point exactly. It is the same we do [currently in Android](https://github.com/customerio/customerio-android/blob/main/messagingpush/src/main/AndroidManifest.xml#L9-L15). And a service added by customer app will always take priority over it I think?

Reply:
> if the customer forgets to set the android:priority property or gives it an incorrect value

That is definitely a valid concern, but the problem is all other services are providing `priority`. So now, in case the customer app doesn't mention any service in the manifest or doesn't mention priority either, some other push services they have also integrated would get control over it and our SDK won't.

Reply:
I understand this, and we might need to discuss about priority. But how does it affect customers when our [Android SDK already does this](https://github.com/customerio/customerio-android/blob/main/messagingpush/src/main/AndroidManifest.xml#L9-L15) and the service eventually added by our SDK with or without this PR would still be added with `android:priority="-1"`?

Reply:
Also, I'm not really sure how does Android handles this for our case. Looking at [Android docs](https://developer.android.com/guide/topics/manifest/intent-filter-element#priority), this value doesn't seem to be adding any value in case of services. If any of you knows how android treats priority for such cases, please let me know.

Reply:
haha, yup it looks like it does the opposite apparently. I think the reason everyone puts it as `-1` is to let the system know this service should have a lower priority than most other Services in the system.